src/Http/Requests/StoreCategory.php

```diff
@@ -25,6 +25,10 @@ public function rules(): array
 
     public function fulfill()
     {
-        return Category::create($this->validated());
+        $category = Category::create($this->validated());
+
+        event(new UserStoredCategory($this->user(), $category));

```

Comment:
Why `UserStoreCategory` when the method called is `Category::create`? Feels an odd name as the user's action isn't to store, the user creates - the byproduct is that it gets stored.